### PR TITLE
fix(@ngtools/webpack): improve missing TS file error message

### PR DESCRIPTION
**Pull request by filipesilva**
Related to https://github.com/angular/angular-cli/issues/8284#issuecomment-341417325

### CODE REVIEW

File: packages/@ngtools/webpack/src/angular_compiler_plugin.ts [hunk 1]
```diff
@@ -816,16 +816,19 @@ export class AngularCompilerPlugin implements Tapable {
...
+      // Check if the TS input file and the JS output file exist.
+      if ((fileName.endsWith('.ts') && !this._compilerHost.fileExists(fileName, false))
+        || !this._compilerHost.fileExists(outputFile, false)) {
+        let msg = `${fileName} is missing from the TypeScript compilation. `
+          + `Please make sure it is in your tsconfig via the 'files' or 'include' property.`;
+
+        if (/(\\|\/)node_modules(\\|\/)/.test(fileName)) {
+          msg += '\nThe missing file seems to be part of a third party library. '
+            + 'TS files in published libraries are often a sign of a badly packaged library. '
+            + 'Please open an issue in the library repository to alert its author.';
```
Comment (IgorMinar) [inline packages/@ngtools/webpack/src/angular_compiler_plugin.ts]: "... and ask them to package the library using the Angular Package Format (https://goo.gl/AfWtpq)."

In that doc I added links to ng-packgr and generator-angular2-library

Comment (filipesilva) [reply packages/@ngtools/webpack/src/angular_compiler_plugin.ts]: Done

**Comment by IgorMinar** [comment]
lgtm. thanks

File: packages/@ngtools/webpack/src/angular_compiler_plugin.ts [hunk 2]
```diff
@@ -816,16 +816,20 @@ export class AngularCompilerPlugin implements Tapable {
...
+      if ((fileName.endsWith('.ts') && !this._compilerHost.fileExists(fileName, false))
+        || !this._compilerHost.fileExists(outputFile, false)) {
+        let msg = `${fileName} is missing from the TypeScript compilation. `
+          + `Please make sure it is in your tsconfig via the 'files' or 'include' property.`;
+
+        if (/(\\|\/)node_modules(\\|\/)/.test(fileName)) {
+          msg += '\nThe missing file seems to be part of a third party library. '
+            + 'TS files in published libraries are often a sign of a badly packaged library. '
+            + 'Please open an issue in the library repository to alert its author and ask them '
+            + 'to package the library using the Angular Package Format (https://goo.gl/AfWtpq).';
```
Comment (devoto13) [inline packages/@ngtools/webpack/src/angular_compiler_plugin.ts]: Link leads to a **not** public document.

Comment (filipesilva) [reply packages/@ngtools/webpack/src/angular_compiler_plugin.ts]: @IgorMinar ^

**Comment by IgorMinar** [comment]
uh oh. let me take a look

**Comment by IgorMinar** [comment]
ok. that was a wrong link. Please use https://goo.gl/jB3GVv thanks!

**Comment by filipesilva** [comment]
Updated link.

**Pull request merged**